pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -26,6 +26,11 @@ in stdenv.mkDerivation rec {
     sha256 = "1p2ncvjz6alr9n3l5wvq2arqgc7xjs6dqyar1l9jp0z8cfgapkb8";
   };
 
+  patches = [
+    # ./config-after-first-failure.patch
```

Comment:
Is this a misstake?

Reply:
Read the description, it's a reference implementation, as well as the root MR request, https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/288#note_672321

I'm not planning on ever getting this into shape, I can't tell any functionality difference between this over the normal pulseaudio, actually causes more instability for me.

Reply:
I will try to work on this, if I find time. One benefit would be having support for absolute volume levels, especially in conjunction with [!239](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/239#note_479656). Another would be bluetooth battery indicator but that still needs upower support.